Dockerfile

```
@@ -1,8 +1,22 @@
-FROM ubuntu:jammy 
+FROM datajoint/pydev:python3.6
+
+RUN apt update && apt -y install mysql-client-5.7 netcat
+
 ENV TEAM="Seahawks"
-COPY ./README.md /home/ubuntu/files/README.md
-ADD https://raw.githubusercontent.com/datajoint/datajoint-python/master/CHANGELOG.md /home/ubuntu/files/CHANGELOG.md
-RUN echo HEY! > /home/ubuntu/files/file1.txt
-USER games:games
+
+RUN mkdir /seahawks
+
+WORKDIR /seahawks
+
+RUN git clone https://github.com/davidgodinez/seahawks-newsletter-pipeline.git /seahawks
+
+RUN pip install --upgrade pip && pip install .
+
+RUN pip install -r requirements.txt
+
+COPY ./dj_local_conf.json /seahawks/dj_local_conf.json
```

When you are copying into a directory and you'd like to preserve the same name, you can use the following as short-hand:
```suggestion
COPY ./dj_local_conf.json /seahawks/
```

---

This line will ultimately be removed since we are copying the directory rather than git cloning.